### PR TITLE
Theming: basic font mixins and single theming SASS export

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

1) Create basic font variables, for the products that already inherit font-family by default.
2) Make one file that exports ALL theming SASS variables, so you only need to every import that one file.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8143)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/common/_themeOverrides.scss [hunk 1]
```diff
@@ -24,7 +24,6 @@ $ms-color-neutralLighterAlt: "[theme:neutralLighterAlt, default: #f8f8f8]";
 $ms-color-accent: "[theme:accent, default: #0078d4]";
 $ms-color-white: "[theme:white, default: #ffffff]";
 $ms-color-whiteTranslucent40: "[theme:whiteTranslucent40, default: rgba(255,255,255,.4)]";
-$ms-color-yellowDark: "[theme:yellowDark, default: #d29200]";
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/common/_themeOverrides.scss]: It looks like `yellowDark` was added in this PR https://github.com/OfficeDev/office-ui-fabric-react/pull/7976 for fluent folder styling. Should we add it to IPalette?

File: packages/office-ui-fabric-react/config/pre-copy.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
       "./src/common/_themeOverrides.scss",
       "./src/common/_legacyThemePalette.scss",
       "./src/common/_effects.scss",
-      "./src/common/_themeVariables.scss"
+      "./src/common/_themeVariables.scss",
+      "./src/common/ThemeVariables.scss"
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/config/pre-copy.json]: I don't see ThemeVariables.scss anywhere, am I missing it?

File: packages/office-ui-fabric-react/src/common/_semanticSlots.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/** THIS FILE IS AUTOGENERATED do not modify it manually. See generateDefaultThemeSassFiles.js. New slots should be added to the appropriate interfaces and defaults files. */
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: Nice!

**Comment by ecraig12345** [comment]
Looks like you still need @dzearing to approve something, not sure what

**Comment by natalieethell** [comment]
Just FYI - @Vitalius1 has a PR open to update some of the palette colors here: https://github.com/OfficeDev/office-ui-fabric-react/pull/8143. Just want to make sure that if this PR is merged after his, we update the files again accordingly.

**Comment by phkuo** [comment]
> Just FYI - @Vitalius1 has a PR open to update some of the palette colors here: #8143. Just want to make sure that if his PR is merged before this one, we update the files again accordingly.

You mean #8148 

@dzearing @dzearing @dzearing

**Comment by ecraig12345** [comment]
Is tagging three times the magical way to summon someone now? 🧙‍♂️

**Comment by natalieethell** [comment]
Oh, just kidding! @Vitalius1 's PR is going into the fabric-7 branch. So I thiiiink we just need to make sure that that branch is updated accordingly following this change.

**Comment by phkuo** [comment]
@cliffkoh helppppppp

**Pull request merged**